__live-tests__/credit-memo-api.test.ts

```
@@ -0,0 +1,106 @@
+// Imports
+import { AuthProvider, Environment, ApiClient, AuthScopes, type CreditMemoOptions } from '../src/app';
+import { describe, expect, test } from 'bun:test';
+
+// Describe the CreditMemo API
+describe('Live API: CreditMemos', async () => {
+	// Initialize the Auth Provider
+	const authProvider = new AuthProvider(process.env.QB_CLIENT_ID!, process.env.QB_CLIENT_SECRET!, process.env.REDIRECT_URI!, [
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)